template/build/webpack.base.conf.js

```diff
@@ -58,7 +58,7 @@ module.exports = {
           postcss: cssUtils.postcss,
           loaders: merge({js: 'babel-loader'}, cssUtils.styleLoaders({
             sourceMap: useCssSourceMap,
-            extract: env.prod
+            extractCSS: env.prod
```

Comment:
This is part of an object which goes into styleLoaders method from cssUtils (/build/css-utils.js).
Take a look here. https://github.com/quasarframework/quasar-template-default/blob/master/template/build/css-utils.js#L32